dev/build_release.py

```diff
@@ -371,11 +371,26 @@ def __gcb_build(cls, name, gradle_root, gcb_service_account, gcb_project):
     if os.path.isdir(gradle_cache):
       # Tell rmtree to delete the directory even if it's non-empty.
       shutil.rmtree(gradle_cache)
-    cmds = [
-      ('gcloud container builds submit --log-http'
-       ' --account={account} --project={project} --config="../{name}-gcb.yml" .'
-       .format(name=name, account=gcb_service_account, project=gcb_project))
-    ]
+    if self.__options.gcb_externalize_gcs_upload:
+      tar_filename = name + '-{:%Y%m%d%H%M%S}'.format(datetime.datetime.utcnow())
+      gcs_staging_bucket = self.__options.gcb_gcs_staging_bucket
+      cmds = [
+        ('tar -cf {tar_filename}.tar .'
+          .format(tar_filename=tar_filename)),
+        ('gcloud config set account {account}'
```

Comment:
I dont think you should do this. It's up to the caller (e.g. jenkins script)
This is a global system wide side effect.
Youd definitely dont want to do that -- it woudl affect other independent jobs running.

I was just pointing out the potential vulnerability in the script.
I dont know how to fix it. It is a gsutil limitation.

Reply:
Is it avoidable? How else will gsutil get credentials to do the upload?

Reply:
Done, removed the set account call.